docs/plugins/autolabor.rst

```diff
@@ -45,10 +45,11 @@ and manager.
 Hunting is never assigned without a butchery, and fishing is never assigned
 without a fishery.
 
-For each labor, a preference order is calculated based on skill, biased against
-masters of other trades and excluding those who can't do the job. The labor is
-then added to the best <minimum> dwarves for that labor, then to additional
-dwarfs that meet any of these conditions:
+For each labor, a preference order is calculated based on skill, excluding those
+who can't do the job. Dwarves who are masters of particular skills are
+deprioritized in the preference list for other skills. The labor is then added
+to the best <minimum> dwarves for that labor, then to additional dwarfs that
+meet any of these conditions:
```

Comment:
I am not sure this is better. And I've got to presume you know how this script is implemented because I had assumed the exclusion was related to the bias against masters part due to the comma (if so, then this change would be incorrect [I think]).

But more to the point "deprioritized in the preference list for other skills" is just as terse.

So long as I comprehend the intent, I'd go with: "Dwarves who are masters of a skill are deprioritized for other skills." Or something like that.

Reply:
fixed. Thanks!

Reply:
It might even be inferred by saying the reverse: "Masters of a skill are prioritized for that skill's labor"